src/lib.rs

```diff
@@ -184,9 +184,10 @@ impl EguiMq {
 
         let egui::FullOutput {
             platform_output,
-            repaint_after: _, // miniquad always runs at full framerate
             textures_delta,
             shapes,
+            pixels_per_point, //TODO not handling change in pixels per point (check zooming in/out behavior in egui)
```

Comment:
This is egui telling you what its (current) definition of a "point" is. It is the value that should be passed to `tessellate`